.travis.yml

```diff
@@ -1,7 +1,7 @@
 # It may be tempting to add parens around each individual clause in this expression, but Travis then builds pushes anyway
 if: branch = master OR branch =~ ^release/ OR branch =~ ^features/ OR tag IS present
 language: go
-go: 1.12.x
+go: 1.14.x
```

Comment:
FWIW, nothing else is upgraded to v1.14.x yet... that will start next week :)

Reply:
I'll switch this back to `1.13.x` in the next push

Reply:
1.13 appears to exhibit different behaviour with regards to building modules from outside of their directory (or at least I thought that's what it did last night) - if we're upgrading to 1.14 this week anyway it might be better to use this as a test bed?